src/views/visual/visual-view-graph.js

```diff
@@ -119,6 +119,30 @@ export class VisualViewGraph extends LitElement {
       </div>
     `;
   }
+  // the following could also be done on the backend, but I am not entirely sure. it is a complex question...
```

Comment:
Indeed best done in the backend.
But why delete the numbers on the right hand side? Very difficult for me to see which collection I'm looking at.
What are you trying to achieve? I notice that on this branch in local it is different from buddhanexus.net

Reply:
it reduces the number of collections and makes the display managable for the Chinese. In the Chinese canon (contrary to the Tibetan), for example Abhidharma is split into T26,T27,T28 etc. The plan here is to just sum them up into larger baskets, which means less nodes on the visualization and a cleaner display.

Reply:
however, I believe that on the level of data-organization, we better stick to the layout of the CBETA and just do this for the visualization. I played a bit on the backend to make these changes already when reading in the dataset but that's a hopeless (= horrifying) undertaking and I don't want to try that again. It introduces all kinds of problems. But we can probably do this in python after we receive the data from the query and before we dispatch it to the API.